app/controllers/tezos/bakers_controller.rb

```diff
@@ -8,6 +8,12 @@ def index
     @tezos_bakers = @tezos_bakers.where("LOWER(id) = ? OR name ILIKE ?", params[:query].downcase, "%#{params[:query]}%") if params[:query]
   end
 
+  # GET /tezos/bakers/count.json
+  def count
+    @chain = Tezos::Chain.primary
```

Comment:
Why set @chain and then @count?  Don't seem to use @chain.

Reply:
Yeah I meant to change that. Good catch